examples/basic-globalize-compiler/README.md

```diff
@@ -0,0 +1,26 @@
+# Basic Globalize Compiler example
+
+On this example, we assume you know the Globalize usage basics. So, it focuses on
```

Comment:
Maybe change these two sentences to:
"This example focuses on the Globalize Compiler and the Globalize runtime modules. It assumes knowledge of Globalize usage basics."